hydra-node/test/Hydra/Model.hs

```diff
@@ -696,6 +743,33 @@ performClose party = do
       err@CommandFailed{} -> error $ show err
       _ -> False
 
+performFanout :: (MonadThrow m, MonadAsync m, MonadDelay m, MonadTimer m) => Party -> RunMonad m UTxO
+performFanout party = do
+  nodes <- gets nodes
+  let thisNode = nodes ! party
+  waitForClosed thisNode
```

Comment:
Should: not wait for closed, but for `ReadyToFanout`

Don't we see model failures where we try to fanout "too early" if we only wait for `HeadIsClosed` message?